flowpipe/graph.py

```diff
@@ -216,7 +216,7 @@ def accepts_connection(self, output_plug, input_plug):
         return True
 
     def evaluate(self, mode="linear", skip_clean=False,
-                 submission_delay=0.1, max_workers=None):
+                 submission_delay=0.1, max_workers=None, data_persistant=True):
```

Comment:
I would suggest calling the parameter `data_persistence` to keep in line with substantives as parameter names as far as reasonably possible.

Reply:
Good point, that is done now